docs-source/content/userguide/managing-fmw-domains/soa-suite/Monitor-Exporter-setup.md

```diff
@@ -0,0 +1,286 @@
+[INFO] skip non existing resourceDirectory <Working_dir>/weblogic-monitor/weblogic-monitoring-exporter/webapp/src/main/resources
+[INFO]
+[INFO] --- maven-compiler-plugin:3.1:compile (default-compile) @ exporter-webapp ---
+[INFO] No sources to compile
+[INFO]
+[INFO] --- maven-resources-plugin:2.6:testResources (default-testResources) @ exporter-webapp ---
+[WARNING] Using platform encoding (UTF-8 actually) to copy filtered resources, i.e. build is platform dependent!
+[INFO] skip non existing resourceDirectory <Working_dir>/weblogic-monitor/weblogic-monitoring-exporter/webapp/src/test/resources
+[INFO]
+[INFO] --- maven-compiler-plugin:3.1:testCompile (default-testCompile) @ exporter-webapp ---
+[INFO] No sources to compile
+[INFO]
+[INFO] --- maven-surefire-plugin:2.12.4:test (default-test) @ exporter-webapp ---
+[INFO] No tests to run.
+[INFO]
+[INFO] --- maven-war-plugin:2.6:war (default-war) @ exporter-webapp ---
+[INFO] Packaging webapp
+[INFO] Assembling webapp [exporter-webapp] in [<Working_dir>/weblogic-monitor/weblogic-monitoring-exporter/webapp/target/wls-exporter]
+[INFO] Processing war project
+[INFO] Copying webapp webResources [<Working_dir>/weblogic-monitor/weblogic-monitoring-exporter/webapp/src/main/notices] to [<Working_dir>/weblogic-monitor/weblogic-monitoring-exporter/webapp/target/wls-exporter]
+[INFO] Copying webapp resources [<Working_dir>/weblogic-monitor/weblogic-monitoring-exporter/webapp/target/configuration]
+[INFO] Webapp assembled in [57 msecs]
+[INFO] Building war: <Working_dir>/weblogic-monitor/weblogic-monitoring-exporter/webapp/target/wls-exporter.war
+[INFO] ------------------------------------------------------------------------
+[INFO] BUILD SUCCESS
+[INFO] ------------------------------------------------------------------------
+[INFO] Total time:  3.621 s
+[INFO] Finished at: 2019-06-20T09:01:12-07:00
+```  
+
+The generated package will be at weblogic-monitoring-exporter/webapp/target/wls-exporter.war.  
+
+## Deploy Weblogic-monitor-exporter  
+
+Deploy weblogic-monitor-exporter in the AdminServer and Soa maanaged servers separately through Oracle Enterprise Manager.   
+The URL would be like http://server:30305/em  
+Here 30305 is the frontend port of the traefik load balancer.
+
+![Deploy monitor exporter](images/Monitor-Exporter_deploy-weblogic-monitor.png)  
+
+Select the servers to deploy:  
+
+![Select servers to deploy](images/Monitor-Exporter_select-servers-to-deploy.png) 
+
+Set the application name:  
+
+The application name has to be different if deployed separately in AdminServer and Soa managed servers.  
+Set the context-root to "wls-exporter" for both the deployments. Select "Install and start application" option.
+
+![Set the application name](images/Monitor-Exporter_set-the-applications-name.png) 
+
+Then deploy weblogic-monitor-exporter application and then activate changes for the application to start.  
```

Comment:
@rosemarymarano, could you please take a look at the sections below and provide guidelines on how to bring the diction and style in line with what we have elsewhere?